### PR TITLE
Use maxFramesPerPage instead of maxBuffersPerPage

### PR DESCRIPTION
**Pull request by Kukunin**
The main reason behind this change is the situation when a size of the device's buffer lesser than a frame size, the library sends empty ogg pages with no data in them.

For example, with buffer size 256 which equals to about 6ms of time, maxBuffersPerPage: 1 and frameSize: 10ms the library sends 172 (1000 / 6) packets per second, and 72 are empty.

Having the buffer size lesser than the frame size improve latency on streaming

### CODE REVIEW

File: src/recorder.js [hunk 1]
```diff
@@ -26,6 +28,11 @@ var Recorder = function( config ){
     streamPages: false,
     wavBitDepth: 16,
   }, config );
+
+  if('maxBuffersPerPage' in config && !('maxFramesPerPage' in config)) {
```
Comment (chris-rudmin) [inline src/recorder.js]: I'm not particularly interested in maintaining backwards compatibility. We will release a new major version when there are config changes.

**Comment by chris-rudmin** [comment]
@Kukunin I see where the fix is now.  Could you add a unit test for this change?

**Comment by Kukunin** [comment]
Good. I'll do, and will remove the backward compatiblity

**Comment by Kukunin** [comment]
@chris-rudmin please check it out

**Comment by chris-rudmin** [comment]
Thanks!

**Pull request merged**